charts/kubernikus-system/templates/_prometheus.yml.tpl

```diff
@@ -8,15 +8,13 @@ global:
   external_labels:
     region: {{ .Values.global.region }}
 
-{{- if .Values.use_alertmanager }}
```

Comment:
What is the reason this isn't needed anymore?

Reply:
We’re always using the alertmanager. Could also leave the condition and default to `true`